tests/extensions/test_manager.py

```diff
@@ -0,0 +1,14 @@
+from saleor.core.extensions.manager import BaseManager, get_extensions_manager
+from saleor.core.extensions.plugin import BasePlugin
+
+
+class TestPlugin(BasePlugin):
+    """"""
+
+
+def test_get_extensions_manager():
```

Comment:
Do we have any plans for testing if calling any plugin methods actually works?

Reply:
Yeap. that was my fault. After I've changed architecture for Plugin and Manager I've removed previous tests but I didn't add any new one.